bin/setup-local-env.sh

```diff
@@ -16,8 +16,8 @@ cd "$(dirname "$0")/.."
 . "$(dirname "$0")/install-docker.sh"
 
 ! read -d '' GUTENBERG <<"EOT"
-,⁻⁻⁻.       .                 |
-|  _. .   . |---  ,---. ,---. |---. ,---. ,---. ,---.
+,⁻⁻⁻·       .                 |
+|  ،~’.   . |---  ,---. ,---. |---. ,---. ,---. ,---.
```

Comment:
The standard tilde is perhaps a bit too harsh. What do you think of "Swung Dash" `⁓` (U+2053)?